data/json/effects.json

```diff
@@ -139,7 +139,8 @@
         "type": "effect_type",
         "id": "shrieking",
         "name": ["Shrieking"],
-        "desc": ["AI tag used for screecher sounds.  This is a bug if you have it."]
+        "desc": ["AI tag used for screecher sounds.  This is a bug if you have it."],
+        "show_in_info": true
```

Comment:
It's also can be used in npc description cause now there is now info about npc effects.